pkg/tid/cli/display/table.go

```diff
@@ -29,7 +30,7 @@ func WriteEntriesTable(entries []types.Entry, writer io.Writer) {
 			entry.Created.Format(entry.CreatedTimeFormat()),
 			entry.Updated.Format(entry.UpdatedTimeFormat()),
 			entry.Note,
-			entry.Duration.String(),
+			getTimeInRightFormat(entry.Duration, config.Display.TimeFormat),
```

Comment:
Perhaps `formatDuration` would be more succinct, and accurate given it's a duration it's formatting, not a time ;)